polybar-scripts/vpn-wireguard-wg/vpn-wireguard-wg.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+CONFIG_PATH=~/wg/wireguard.conf
+
+SHOW_NAME=false #Show connection name instead of CONNECTED_TEXT
+
+CONNECTED_ICON="#1 VPN:"
+CONNECTED_TEXT="up"
+
+DISCONNECTED_ICON="#2 VPN:"
+DISCONNECTED_TEXT="down"
+
+is_exists() {
+    if [ ! -f $CONFIG_PATH ]; then
+        echo "$DISCONNECTED_ICON Config file not found"
+        exit 0
+    fi
+}
+
+check() {
+    CONFIG_NAME=$(basename "${CONFIG_PATH%.*}")
```

Comment:
Your last commit was a nice idea. 
You could move this two lines outside this function and aftr that you can simplify the connected condition too.

Reply:
I don't think it's a good idea because connected condition is changing in `toggle()`, so if we don't want to get double delay for status update, we need recheck connection and only after that call `status()`.

Reply:
Ah, yes. You are right.

Reply:
> Ah, yes. You are right.

I thought, that polybar is updating module text on click action... It doesn't - https://github.com/jaagr/polybar/issues/786
But as I understand this feature will be added in the next update - [click me](https://github.com/jaagr/polybar/projects/1#card-5907583)
So maybe left that as is instead of updating on next polybar release?